packages/terra-clinical-item-view/src/terra-dev-site/doc/clinical-item-view/clinicalItemView.1.doc.mdx

```diff
@@ -37,6 +38,10 @@ import ItemView from 'terra-clinical-item-view';
     title="ItemView - Two Column Layout"
     description="When including icons as part of an '<ItemView.Display />'', it is recommended to use the iconAlignment='inline' prop for best alignment and wrapping with the text."
 />
+<ItemViewTwoColumnByRow
+    title="ItemView - Two Column Layout By Row"
+    description="When the `trueColumn` prop is set to `false` the displays are split two at a time into rows which are stylized to look like columns instead of true columns. So the ItemView is formatted to look like columns but programatically it's handled by row."
```

Comment:
nit: Same for the `programmatically`

Reply:
[400fc6d](https://github.com/cerner/terra-clinical/pull/912/commits/400fc6dbe82c75e34d3d09ec6d5ce2e191aa2a2c)